### PR TITLE
Closed PR

### PR DESCRIPTION
**Pull request by hinosxz**
Closed

### CODE REVIEW

**Review by cnharris10 — APPROVED** [review]
Policy logic looks good.  Don't know enough about CloudFormation setup to address YAML settings though

File: aws/datadog_policy_macro.yaml [hunk 1]
```diff
@@ -65,6 +65,48 @@ Resources:
...
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:GetObject"
+                    ],
+                    "Resource": data_path
+                }
+            ]
+
+            if has_cost_explorer_permissions:
```
Comment (ashwinjv) [inline aws/datadog_policy_macro.yaml:94]: I believe `has_cost_explorer_permissions` is a string, with allowed values `true`, `false` and not a bool

File: aws/datadog_policy_macro.yaml [hunk 2]
```diff
@@ -65,6 +65,48 @@ Resources:
...
+                        "ce:Get*"
+                    ],
+                    "Resource": "*"
+                })
+
+            return {
+                "PolicyName": "DatadogCloudCostPolicy",
+                "PolicyDocument": {
+                    "Version": "2012-10-17",
+                    "Statement":
```
Comment (ashwinjv) [inline aws/datadog_policy_macro.yaml:107]: I think you are missing the `statement` here?

File: aws/datadog_policy_macro.yaml (hunk 1 above)
Comment (ashwinjv) [reply aws/datadog_policy_macro.yaml:94]: Also, should we only look to install the `DatadogCloudCostPolicy` if `has_cost_explorer_permissions` is True? I see that we do return a `AWS::NoValue` if bucket_name/report_name is empty.

**Pull request closed without merging**